docs/source/pipeline_tutorial.mdx

```diff
@@ -0,0 +1,130 @@
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Pipeline
+
+The [`pipeline`] makes it simple to use any model from the [Model Hub](https://huggingface.co/models) for inference. You can use the [`pipeline`] for a variety of tasks such as text generation, image segmentation and audio classification. Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`pipeline`]! This tutorial will teach you to:
+
+* Use a [`pipeline`] for inference.
+* Use a specific tokenizer or model.
+* Use a [`pipeline`] for audio and vision tasks.
+
+<Tip>
+
+See [here](main_classes/pipelines) for a complete list of tasks supported by [`pipeline`].
+
+</Tip>
+
+## Pipeline usage
+
+While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the specific task pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of completing your task. 
+
+1. Start by creating a [`pipeline`] with your desired inference task:
+
+```py
+>>> from transformers import pipeline
+
+>>> generator = pipeline(task="text-generation")
+```
+
+2. Pass your input text to the [`pipeline`]:
+
+```py
+>>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+[{'generated_text': 'Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone, Seven for the Iron-priests at the door to the east, and thirteen for the Lord Kings at the end of the mountain'}]
+```
+
+If you have more than one input, pass your input as a list:
+
+```py
+>>> generator([
+    "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone",
+    "Nine for Mortal Men, doomed to die, One for the Dark Lord on his dark throne"
+])
+```
+
+Any additional parameters to your task can also be included in the [`pipeline`]. For example, `text-generation` has a [`generation_utils.GenerationMixin.generate`] method with several parameters for controlling the output. Here, you can set the `num_return_sequences` parameter to return more than one output:
+
+```py
+>>> generator(
+    "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone",
+    num_return_sequences=2,
+)
+```
+
+### Choose a model and tokenizer
+
+The [`pipeline`] accepts any model from the [Model Hub](https://huggingface.co/models). You can use with the tags on the Model Hub to filter for a model you'd like to use for your task. Then you can load a model and tokenizer with the [`AutoModel`] and [`AutoTokenizer'] classes:
```

Comment:
Ah I see, instead of referring directly to the [`AutoModel`](https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoModel) class, what we really mean is an `AutoModelForX`!